docker/Dockerfile-lb

```diff
@@ -0,0 +1,15 @@
+FROM gliderlabs/alpine
+
+RUN echo "@testing http://dl-4.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories
+
+RUN apk-install nginx curl runit@testing
+
+RUN curl -L -o /tmp/consul-template.zip http://releases.hashicorp.com/consul-template/0.12.0/consul-template_0.12.0_linux_amd64.zip
+RUN unzip /tmp/consul-template.zip -d /usr/local/bin
+
+ADD nginx.service /etc/service/nginx/run
+ADD consul-template.service /etc/service/consul-template/run
+
+ADD nginx.conf /etc/consul-templates/nginx.conf
+
+CMD ["/sbin/runsvdir", "/etc/service"]
```

Comment:
[`runsvdir`](http://smarden.org/runit/runsvdir.8.html) here is starting and monitoring the nginx and consul-template services we previously added to `/etc/service`